src/apps/material/material.dev.tsx

```diff
@@ -536,6 +536,17 @@ export default {
       name: "Url where user can authenticate",
       defaultValue: "",
       control: { type: "text" }
+    },
+    skipQueueMaterialIsAvailableInAnotherEditionText: {
+      name: "Skip queue material is available in another edition",
+      defaultValue:
+        "Skip the queue - The material is available in another edition",
+      control: { type: "text" }
+    },
+    skipQueueMaterialIsAvailableInAnotherEditionReservationsText: {
+      name: "Skip queue reservations cont",
```

Comment:
Do not understand name. What does it mean?

Reply:
I have changed it to materialIsAvailableInAnotherEditionReservationsText 😊